recipes/minimap2/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: f4c8c3459c7b87e9de6cbed7de019b48d9337c2e46b87ba81b9f72d889420b3c
 
 build:
-  number: 0
+  number: 1
```

Comment:
Why is this failing lint? The build number is incremented relative to master: https://github.com/bioconda/bioconda-recipes/blob/af04e627ab11092e31545d965dbca42704b2ca80/recipes/minimap2/meta.yaml#L12

Reply:
https://anaconda.org/bioconda/minimap2/files

We have rebuilt a new version on bulk I think in the meantime - which is not yet merged back to main.